client/src/components/common/names-block.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-row align="start" class="mx-0 bg-light-gray">
+    <v-col class="text-name pa-4">
+      <div
+        v-for="name of names"
+        :key="`name-${name.choice}`"
+        :class="getClass(name)"
+      >
+        {{ `${name.choice}. ${name.name}` }}
+        <v-icon
+          v-if="getIcon(name)"
+          class="mt-n1"
+          :class="getClass(name)"
+        >
+          {{ getIcon(name) }}
+        </v-icon>
+        <a
+          href="#"
+          class="link-sm ml-1"
+          v-if="name.state === NameState.CONDITION"
+          @click.prevent="onConditionsClicked()"
+        >Conditions</a>
+      </div>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { NameState } from '@/enums'
+
+@Component({})
+export default class NamesBlock extends Vue {
```

Comment:
Where do we use this? just for a little context

Reply:
This is the names block sub-component at the top of the Existing Request Display page. It looks like this:

![image](https://user-images.githubusercontent.com/10002889/100691206-0aac0780-333d-11eb-9095-cb1906eb9206.png)

Reply:
PS I just moved this sub-component from the common folder to the existing request folder.